### PR TITLE
Create UCIAtlas.yaml

### PR DESCRIPTION
**Pull request by djw8605**
Did I do this right?

### CODE REVIEW

File: projects/UCIAtlas.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# FieldOfScience is a more specific description, e.g. "High Energy Physics"
+FieldOfScience: High Energy Physics
+# Organization is the organization where the project is hosted, e.g. "UCSD"
+Organization: UCI
+# PIName is the prinicipal investigator for the project
+PIName: Anyes Taffard
+
+# If you have an up-to-date local git clone, fill ID with the output from `bin/next_project_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+ID:
```
Comment (matyasselmeci) [inline projects/UCIAtlas.yaml]:
```suggestion
ID: 537
```

File: projects/UCIAtlas.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Sponsor:
+  ### Uncomment the CampusGrid block if the project was sponsored by a campus grid e.g. OSG Connect
```
Comment (matyasselmeci) [inline projects/UCIAtlas.yaml]: You need a sponsor here. Since it's atlas, perhaps
```
Sponsor:
  VirtualOrganization:
    Name: ATLAS
```

**Comment by djw8605** [comment]
I don't understand what's wrong.  Any ideas?

File: projects/UCIAtlas.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  # CampusGrid:
+  ##### Name is the specific campus grid that sponsored the project
+  #   Name: <NAME>
+
+  ### Uncomment the VirtualOrganization block if the project was sponsored by a VO
+  VirtualOrganization:
+  ##### Name is the name of the VO that sponsored the project
+    Name: ATLAS
```
Comment (matyasselmeci) [inline projects/UCIAtlas.yaml]:
```suggestion
    ID: 35
    Name: ATLAS
```
Looking at the other projects, it looks like you need the VO ID as well, not just the name. I think this is a bug.

Comment (djw8605) [reply projects/UCIAtlas.yaml]: Makes sense in the DB world.  Linking things together...

**Pull request merged**